src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -389,6 +396,9 @@ export class SavedObjectsService
         }
         this.savedObjectServiceCustomStatus$ = status$;
       },
+      addExtendedSavedObjectsMappings: (mappings: SavedObjectsMappingProperties) => {
+        this.extendedSavedObjectsMappings = mappings;
+      },
```

Comment:
could be a list, so that other plugin could add extended mapping.

Reply:
Makes sense, I'll change that.